package.json

```diff
@@ -7,11 +7,13 @@
     "npm": ">=3"
   },
   "scripts": {
+    "cover": "mastarm test --coverage --coverage-paths index.js",
+    "lint": "mastarm lint",
     "prepublish": "mastarm prepublish index.js:build/index.js",
     "prestart": "yarn",
-    "test": "mastarm lint",
     "start": "mastarm build --serve example.js",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "test": "yarn run lint && mastarm test"
```

Comment:
`yarn run lint` is 13 characters, `mastarm lint` is 12. There really shouldn't be a need to add a lint command to npm scripts.